hydra-cluster/test/Test/DirectChainSpec.hs

```diff
@@ -57,15 +55,14 @@ spec = around showLogsOnFailure $ do
     bobsCallback <- newEmptyMVar
     withTempDir "hydra-cluster" $ \tmp -> do
       config <- newNodeConfig tmp
-      aliceKeys@(aliceCardanoVk, aliceCardanoSk) <- keysFor Alice
-      withBFTNode (contramap FromCluster tracer) config [aliceCardanoVk] $ \(RunningNode _ nodeSocket) -> do
+      aliceKeys@(aliceCardanoVk, _) <- keysFor Alice
+      withBFTNode (contramap FromCluster tracer) config $ \node@(RunningNode _ nodeSocket) -> do
         bobKeys <- keysFor Bob
-        pparams <- queryProtocolParameters defaultNetworkId nodeSocket
         cardanoKeys <- fmap fst <$> mapM keysFor [Alice, Bob, Carol]
         withIOManager $ \iocp -> do
           withDirectChain (contramap (FromDirectChain "alice") tracer) magic iocp nodeSocket aliceKeys alice cardanoKeys (putMVar alicesCallback) $ \Chain{postTx} -> do
             withDirectChain nullTracer magic iocp nodeSocket bobKeys bob cardanoKeys (putMVar bobsCallback) $ \_ -> do
-              postSeedPayment (Testnet magic) pparams availableInitialFunds nodeSocket aliceCardanoSk 100_000_000
+              void $ seedFromFaucet (Testnet magic) node aliceCardanoVk 100_000_000 Marked
```

Comment:
suggestion: have a `seedFromFaucet_` function to remove the need to `void` it. I am not sure what we use the output for, or even if it's used anywhere.